tests/file_storage/models.py

```diff
@@ -31,8 +32,13 @@ def random_upload_to(self, filename):
         # to make sure it only gets called once.
         return '%s/%s' % (random.randint(100, 999), filename)
 
+    def pathlib_upload_to(self, filename):
+        return Path('bar') / filename
```

Comment:
Can we move tests for callable that returns `pathlib.Path` to a separate commit? It works before this change.